tigerpath/majors_and_certificates/majors/FIT_2018.json

```diff
@@ -0,0 +1,362 @@
+{
+  "type": "Major",
+  "name": "French and Italian",
+  "code": "FIT",
+  "degree": "AB",
+  "year": 2018,
+  "explanation": "The Department of French and Italian offers four tracks for concentrators and a flexible, individualized approach to course planning. Students either pursue a course of study entirely in French or Italian, or combine this specialization with another language, academic discipline, or creative art. FIT encourages innovative and interdisciplinary approaches to the study of the Italian, French, and francophone worlds, tailored to the unique academic and professional interests of students.",
+  "urls": [
+    "https://ua.princeton.edu/academic-units/department-french-and-italian",
+    "https://fit.princeton.edu/undergraduate/concentrating/plan-study",
+    "https://fit.princeton.edu/sites/fit/files/fit_undergraduate_handbook_2019-20_new_0.pdf"
+  ],
+  "contacts": [
+    {
+      "type": "Chair",
+      "name": "Thomas A. Trezise",
+      "email": "ttrezise@princeton.edu"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Christy Wampole",
+      "email": "cwampole@princeton.edu"
+    }
+  ],
+  "req_list": [
+    {
+      "name": "Concentration",

```

Comment:
Were you not able to merge this node with the root?

It's not a big deal either way, and I guess it might be better this way anyway in case we decide to start checking the independent work and thesis requirements.

Reply:
The restructure looks good, by the way.